gui/src/installer/view/editor/template/mod.rs

```diff
@@ -47,6 +48,25 @@ pub fn choose_descriptor_template(progress: (usize, usize)) -> Element<'static,
                 )
                 .width(Length::Fill),
             )
+            .push(
+                card::simple(
+                    Row::new()
+                        .align_items(Alignment::Center)
+                        .push(
+                            Column::new()
+                                .align_items(Alignment::Start)
+                                .push(h3("Multisig security wallet"))
+                                .push(p2_regular("A secure scheme requiring stricter multiparty signature and recovery").style(color::GREY_2))
```

Comment:
nit: The "Simple Inheritance" description above ends with a full stop, but this and the "Custom" description don't.

I would choose without the full stop for now as the descriptions are brief.